bigquery/tests/unit/test_client.py

```diff
@@ -607,6 +665,11 @@ def __init__(self, *responses):
         self._requested = []
 
     def api_request(self, **kw):
+        from google.cloud.exceptions import NotFound
         self._requested.append(kw)
+
+        if len(self._responses) == 0:
```

Comment:
I grabbed this from https://github.com/GoogleCloudPlatform/google-cloud-python/pull/3804